### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by Gallaecio**
To be discussed, including at least @kmike, as I am not sure whether we want to go all in with best practices or keep things more minimal.

### CODE REVIEW

**Comment by Gallaecio** [comment]
I’ve just noticed https://github.com/wRAR/zyte-crawlers-project/pull/1, which should be reviewed and merged before this one is even considered.

File: setup.py [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# A basic setup.py needed for shub deploy to work
```
Comment (wRAR) [inline setup.py:1]: Does it actually work without setup.py?

**Comment by wRAR** [comment]
> I am not sure whether we want to go all in with best practices or keep things more minimal.

Yeah, I'm not sure if we should propose linters or not, mostly.

File: README.rst [hunk 2]
```diff
@@ -2,13 +2,71 @@
...
-* zyte-spider-templates
+
+
+First steps
+===========
+
+After you clone this repository, follow these step to make it yours:
+
+#.  Rename the ``zyte_crawlers_project`` folder to a valid Python module name
+    that you would like to use as your project ID, and update ``scrapy.cfg``
```
Comment (wRAR) [inline README.rst]: Should we say "project name" instead? Every time I see "project ID" I think about the numeric IDs of Scrapy Cloud projects.

File: setup.py (hunk 1 above)
Comment (Gallaecio) [reply setup.py:1]: `shub deploy` generates it automatically, so we could keep it off the repo, and in `.gitignore`.

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[flake8]
```
Comment (kmike) [inline .flake8:1]: I'd keep it out of the scope; it's a bit opinionated to include linters, etc. 

Even requirements.in could be an overkill here, though no strong opinion on that. The issue is that unless people get used to .in/.txt tooling, they'd be getting old packages.

**Comment by Gallaecio** [comment]
Closing, #1 and follow-up PRs covered everything we needed I think.

**Pull request closed without merging**